service/src/main/java/dk/kvalitetsit/hello/service/HelloServiceImpl.java

```diff
@@ -14,10 +17,31 @@ public class HelloServiceImpl implements HelloService {
 
     public HelloServiceImpl(HelloDao helloDao) {
         this.helloDao = helloDao;
+    }   
+
+    @Override
+    public List<HelloServiceOutput> helloServiceGetAll() {
+        var dbEntries = helloDao.findAll();
+        var result = dbEntries.stream()
+            .map(dbEntry -> new HelloServiceOutput(dbEntry.name(), ZonedDateTime.now()))
+            .collect(Collectors.toList());
+
+        return result;
     }
 
     @Override
-    public HelloServiceOutput helloServiceBusinessLogic(HelloServiceInput input) {
+    public List<HelloServiceOutput> helloServiceGetByName(HelloServiceInput input) {
+        var name = input.name();
+        var dbEntries = helloDao.findByName(name);
+        var result = dbEntries.stream()
+            .map(dbEntry -> new HelloServiceOutput(dbEntry.name(), ZonedDateTime.now()))
+            .collect(Collectors.toList());
+        return result;
+    }
+
+
+    @Override
+    public HelloServiceOutput helloServicePost(HelloServiceInput input) {
```

Comment:
Jeg ved godt at det ikke er strengt nødvendigt i det her tilfælde, men jeg synes metoden skal annoteres med `@Transactional` for at indikere at de er sådan man gør hvis der skal laves en databasetransaktion.